test/controllers/landing-page.controller.test.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// For running our service
+const { init } = require('../../app/server.js')
+
+describe('Landing Page controller', () => {
+  let server
+
+  // Create server before each test
+  beforeEach(async () => {
+    server = await init()
+  })
+
+  describe('when a request is made to correct URL', () => {
+    it('displays the landing page', async () => {
+      const options = {
```

Comment:
We always declare any variables we need for tests before the `it` block of the test.
So 
`      options = {
        method: 'GET',
        url: '/landing-page'
      }`
Should be declared under the second describe block.